eclass/base.eclass

```diff
@@ -22,25 +22,27 @@
 # QA Team <qa@gentoo.org>
 # @AUTHOR:
 # Original author: Dan Armak <danarmak@gentoo.org>
-# @SUPPORTED_EAPIS: 0 1 2 3 4 5
+# @SUPPORTED_EAPIS: 4 5
 # @BLURB: The base eclass defines some default functions and variables.
 # @DESCRIPTION:
 # The base eclass defines some default functions and variables.
 
+case "${EAPI:-0}" in
+	[0-3])
+		die "Unsupported EAPI=${EAPI:-0} (too old) for ${ECLASS}"
+		;;
+	[45])
+		;;
+	*)
+		die "${ECLASS}.eclass is banned in EAPI ${EAPI}"
+		;;
+esac
```

Comment:
Again, this was better readable before.

Reply:
you mean the `EXPORT_FUNCTIONS` conditional code?